tests/test_ignite.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+import chainer_pytorch_migration.ignite
+
+
+def test_version_comparisson():
+    version_fnct = chainer_pytorch_migration.ignite._get_version
+    assert version_fnct('0.0.0') == [0, 0]
+    assert version_fnct('0.3.0') == [0, 3]
+    assert version_fnct('0.10.0') == [0, 10]
+    # This is because we only compare up to the minor
+    assert version_fnct('0.3.0') == [0, 3]
+    assert version_fnct('0.3.1a') == [0, 3]
+
+
+def test_invalid_version():
+    version_fnct = chainer_pytorch_migration.ignite._get_version
+    with pytest.raises(ValueError):
+        version_fnct('0.3a.0')
```

Comment:
Why is this invalid?

Reply:
ignite versioning does not use characters in the minor number.
major.minor.patch.identifier

Reply:
The regexp used for semantic versioning also sees that version string as invalid
https://github.com/semver/semver/issues/232

Reply:
So, does this function assume semver numbers?
If so, it should be documented in the implementation.
Also, is it reasonable to assume that? Is that stated in the ignite documentation?

Reply:
Honestly I think we don't have to make this function's assumption very concrete by writing these tests.

Reply:
Decided to leave this as internal so tests are deleted.
Function has been further documented with the assumptions on ignite versioning system.